README.md

```diff
@@ -58,44 +84,25 @@ secrets:
         external: true
 ```
 
+## Development
+
 ### Running Locally
 
 The tool has been designed specifically to run using docker swarm or compose but it can be run locally if required. I have tryed my best to remove all issues when running locally but it may not be perfect.
 
-* First Clone the repository using following the command:
+1. Clone the repository using following the command:
 ```
 https://github.com/uccser/swarm-stack-updater.git
 ```
-* Then use the following command to pull BATS, which is used for automated testing:
+2. Use the following command to pull BATS, which is used for automated testing. This will pull all the required tools for running and creating tests for this application.
```

Comment:
```suggestion
2. Use the following command to pull BATS, which is used for automated testing.
  This will pull all the required tools for running and creating tests for this application.
```